### PR TITLE
DOC: fix symlink to `logistic.sf` in `stats.logistic`

### PR DESCRIPTION
**Pull request by tupui**
Introduced in #15114

### CODE REVIEW

**Comment by ev-br** [comment]
Ugh. The doc build  CI is still unhappy. Would you be able to look into it  @tupui , or should be just revert gh-15114?

**Comment by tupui** [comment]
Ark yes I will do this tomorrow.

**Comment by tupui** [comment]
I think we should just not try to link. We don't do this for other classes when we refer to the PDF of CDF.

**Comment by rgommers** [comment]
Agreed, shouldn't use linking at all. Can you update to double backticks?

File: scipy/stats/_continuous_distns.py [hunk 1]
```diff
@@ -5160,8 +5160,8 @@ class logistic_gen(rv_continuous):
 
     `logistic` is a special case of `genlogistic` with ``c=1``.
 
-    Remark that the survival function (`sf`) is equal to the Fermi-Dirac
-    distribution describing fermionic statistics.
+    Remark that the survival function (`logistic.sf`) is equal to the
```
Comment (ev-br) [inline scipy/stats/_continuous_distns.py:5163]:
```suggestion
    Remark that the survival function (``logistic.sf``) is equal to the
```

**Comment by tupui** [comment]
It's green, I will merge now to make the CI green again. Thanks @ev-br

**Comment by ev-br** [comment]
Thanks Pamphile for fixing this up!

**Pull request merged**